README.md

```diff
@@ -43,6 +43,9 @@ See [example_test.go](https://github.com/fsnotify/fsnotify/blob/master/example_t
 
 ## FAQ
 
+**Are symlinks followed?**
+As of v2.0.0 symlinks are no longer followed. You will have to explicitly get the underlying path with [`filepath.EvalSymlinks(path)`](https://golang.org/pkg/path/filepath/#EvalSymlinks) before passing it to the watcher if you want this behaviour.
```

Comment:
Should we add an example (similar to example_test) of using this?

Reply:
I added a block to `example_test.go` which uses `filepath.EvalSymlinks(path)`, and linked to it here. I'm imagining having it in the standard example is better so people don't miss it.

Reply:
Now there's both `watcher.Add(name)` and `watcher.AddRaw(name)` in the example